test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
By all means, file a PR with the linting errors. Merging things like that is usually a box-ticking exercise that I can do with half of my brain turned off, so I'm not too worried about it. =D